player/lua/ytdl_hook.lua

```diff
@@ -507,8 +507,9 @@ local function add_single_video(json)
 
         if requested_formats then
             for _, track in pairs(requested_formats) do
-                max_bitrate = track.tbr > max_bitrate and
-                    track.tbr or max_bitrate
+                if not (track.tbr == nil) and track.tbr > max_bitrate then
```

Comment:
Forgive me for suggesting such a minor change but...
```suggestion
                if track.tbr ~= nil and track.tbr > max_bitrate then
```

Reply:
Thank you, I have never written a line of lua before. Anyway, this pattern is used throughout the file, should I change all other instances as well?

Reply:
Hm, would be better to leave it consistent then. (Though I don't understand why anyone would choose to write it like that)

Reply:
Both `not (stuff == nil)` and `stuff ~= nil` are used inconsistently in this and osc.lua, iirc.
Both should work the same way, but it might be easier to understand the former.

Reply:
The general rule of thumb with PRs is to only change what needs changing.

If you think the file needs an overhaul to make it more consistent, then this should be in a different PR or commit, but IMO in this case change only what needs changing, and leave the rest of the file as is.